README.md

```diff
@@ -349,6 +350,42 @@ To see a full example of how to start a new project based on this box go to [tem
 
 `NB` **If you are behind a transparent proxy, follow [proxy documentation](#proxy)**
 
+### Custom port forwarding
+
+There is opportunity to forward ports from vagrant box instance to localhost using `vagrant ssh --`
+```text
+vagrant ssh -- --help
+
+usage: ssh [-46AaCfGgKkMNnqsTtVvXxYy] [-B bind_interface]
+           [-b bind_address] [-c cipher_spec] [-D [bind_address:]port]
+           [-E log_file] [-e escape_char] [-F configfile] [-I pkcs11]
+           [-i identity_file] [-J [user@]host[:port]] [-L address]
+           [-l login_name] [-m mac_spec] [-O ctl_cmd] [-o option] [-p port]
+           [-Q query_option] [-R address] [-S ctl_path] [-W host:port]
+           [-w local_tun[:remote_tun]] destination [command]
+```
+
+Example shows how to forward minio(which uses as artifactory) port `:9000` from vagrant box to localhost port `:6666`
+```text
+vagrant ssh -- -L 6666:localhost:9000
+```
+
+> Syntax `vagrant ssh -- -L <local machine port>:<host inside machine>:<port inside machine>`  
```

Comment:
```suggestion
The syntax is

```text
Syntax `vagrant ssh -- -L <local machine port>:<host inside machine>:<port inside machine>`
```

and the example below shows how to forward MinIO (which is used as an artifact manager) port `:9000` from vagrant box to localhost port `:6666`

```text
vagrant ssh -- -L 6666:localhost:9000
```
```

Reply:
Uuuh, not sure what happened with this suggestion?